docs/basics.rst

```diff
@@ -29,101 +29,43 @@ the Git repository.
+
+    dokang.uploaded_docs.dir
+        The path where html documentation will be uploaded.
 
-   dokang.doc_sets
-       The path of the second configuration file (see below) that
-       lists document sets.
+        To define this path, you may use ``%(here)s`` to denote the
+        directory that holds the INI file.
 
-       To define this path, you may use ``%(here)s`` to denote the
-       directory that holds the INI file.
+    dokang.uploaded_docs.token
+        The identification token used to allow documentation upload
+
+    dokang.uploaded_docs.harvester
+        The harvester to use for all projects (full qualified class name)
 
    You may want to start from the example file and only customize
-   these three values. For further details about Pyramid-related
+   these five values. For further details about Pyramid-related
    settings, see `the corresponding section
    <http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/environment.html>`_
    as well as the `Logging
    <http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/logging.html>`_
    section in the Pyramid documentation.
 
-2. A second file (usually called ``doc_sets.py``), lists all document
-   sets with their properties: title, URL, etc. It is a regular Python
-   file that must define a ``DOC_SETS`` sequence, like this:
-
-   .. code:: python
-
-      from dokang.harvesters import sphinx_rtd_config
-
-      DOC_SETS = (
-          {'id': dokang',
-           'title': "Dokang",
-           'path': '/home/docs/dokang/_build/html',
-           'harvester': sphinx_rtd_config(),
-           'url': 'http://docs.exemple.com/dokang',
-          },
-      )
-
-   Each item of ``DOC_SETS`` must be a dictionary with the following
-   keys:
-
-   id
-       The id of the document set. It must be unique in ``DOC_SETS``.
-
-   title
-       The title of the document set. This title is displayed along
-       search results by the command-line client and the web frontend.
-
-   path
-       The local path of the directory that holds the documents.
-
-   harvester
-       A dictionary that holds the configuration of the harvester,
-       which is the component that is responsible of retrieving
-       content and metadata (title) from each file. Dokang ships with
-       a few utilities that provide sane values.
-
-       See :ref:`advanced_harvester_config` in the next chapter for
-       further details.
-
-   url
-       The base URL of the document set. This URL is used by the web
-       frontend. Dokang does **not** serve the indexed documents, it
-       provides only a link to them.
-
-   Each text-like value should be a string (in Python 3) or a unicode
-   object (in Python 2).
-
-   A sample file can be found in the source as `doc_sets.py.sample
-   <https://github.com/Polyconseil/dokang/blob/master/doc_sets.py.sample>`_.
-
-.. note::
-
-   Having two configuration files is a bit unfortunate but it helps
-   running the web frontend.
-
 
```

Comment:
We now need a new "Upload and index documentation" section that explains how to upload documentation to Dokang. As an example, we could include a "curl" or "wget" command line (if possible) or the content of our custom "upload_docs" directive.